lib/grammers-client/src/types/chat/mod.rs

```diff
@@ -107,13 +107,17 @@ impl Chat {
         match packed.ty {
             PackedType::User => {
                 let mut user = User::from_raw(tl::types::UserEmpty { id: packed.id }.into());
-                user.raw.access_hash = packed.access_hash;
+                if let tl::enums::User::User(user) = &mut user.raw {
+                    user.access_hash = packed.access_hash;
+                }
```

Comment:
This would've not worked. I just pushed the proper fix. (And is also why PRs should focus on a single change. Or at least commits should.)